holoviews/operation/datashader.py

```diff
@@ -1742,7 +1742,8 @@ class inspect_mask(Operation):
        Size of the mask that should match the pixels parameter used in
        the associated inspection operation.""")
 
-    streams = param.List(default=[PointerXY])
+    streams = param.ClassSelector(default=dict(x=PointerXY.param.x,
+                                               y=PointerXY.param.y), class_=(dict, list))
```

Comment:
Why change these, the original seemed simpler in this case?

Reply:
Addressed in dbb4ce390